agent/agentssh/agentssh.go

```diff
@@ -272,6 +279,18 @@ func (s *Server) sessionHandler(session ssh.Session) {
 		extraEnv = append(extraEnv, fmt.Sprintf("DISPLAY=:%d.0", x11.ScreenNumber))
 	}
 
+	if s.fileTransferBlocked(session) {
+		s.logger.Warn(ctx, "file transfer blocked", slog.F("session_subsystem", session.Subsystem()), slog.F("raw_command", session.RawCommand()))
+
+		if session.Subsystem() == "" { // sftp does not expect error, otherwise it fails with "package too long"
```

Comment:
for reviewers:

we could experiment and return message, but it requires some [hacking around the sftp server](https://github.com/mafredri/sftp/blob/5bdc2b0e679d0d8336dc63f8b790108667ea4280/server.go#L331).

Reply:
Would it make the detection of this case more reliable?

Reply:
No, it is just about presenting information in the terminal. I haven't confirmed it yet, so it may not be true.